### PR TITLE
Fixes #1364

### LINKED ISSUE #1364: collect user information with bigip_device_info
to help with getting user facts as per: https://github.com/F5Networks/f5-ansible/issues/1362 we need to add bigip user information to bigip_device_info module

### PR DESCRIPTION
**Pull request by nitinthewiz**
I've left out the encrypted password as it doesn't make sense to include that information in the response. Primary use of bigip_device_info -> users  is to allow customers to track the users that exist.

### CODE REVIEW

File: library/modules/bigip_device_info.py [hunk 1]
```diff
@@ -6088,6 +6090,47 @@
       type: bool
       sample: yes
   sample: hash/dictionary of values
+users:
```
Comment (wojtek0806) [inline library/modules/bigip_device_info.py]: maybe also add there what shell the user can access, to be in line with what the bigip_user returns too:

https://github.com/F5Networks/f5-ansible/blob/devel/library/modules/bigip_user.py#L200

File: test/integration/targets/bigip_device_info/tasks/issue-01364.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+
+- name: Issue 01364 - Select users facts
+  bigip_device_info:
+    include:
+      - users
+  register: result
+
+# - name: Issue 00082 - Assert Select interface facts
```
Comment (wojtek0806) [inline test/integration/targets/bigip_device_info/tasks/issue-01364.yaml]: incorrect checks, complete the test, i would suggest, tests like this: create a resource and then get it with facts and verify it is created

Comment (nitinthewiz) [reply test/integration/targets/bigip_device_info/tasks/issue-01364.yaml]: Fixed up the checks. Added the check as you suggested.

File: library/modules/bigip_device_info.py (hunk 1 above)
Comment (nitinthewiz) [reply library/modules/bigip_device_info.py]: Added. I was not sure how the shell works. Now, I'm using it as part of the checks!

**Comment by nitinthewiz** [comment]
Now with 100% fewer whitespaces!

**Pull request merged**